docs/stdlib/abstract.rst

```diff
@@ -4,8 +4,8 @@
 Abstract Types
 ==============
 
-These types are used in definitions of polymorphic (generic) functions
-that can be applied to a broad range of types.
+Abstract types are used to describe a polymorphic, otherwise known as
+generic function to be applicable over a broad range of types.
```

Comment:
New suggestion taking into account @vpetrovykh's feedback:
```suggestion
Abstract types are used to describe polymorphic functions, otherwise known as
"generic functions," which can be called on a broad range of value types.
```